cmd/botkube/main.go

```diff
@@ -394,18 +394,13 @@ func reportFatalErrFn(logger logrus.FieldLogger, reporter analytics.Reporter) fu
 
```

Comment:
Remember about updating the docs afterwards - change configuration of all apps to get rid of the `/botkubehelp` command 👍

Reply:
Docs updated, PTAL: https://github.com/kubeshop/botkube-docs/pull/142